include/seqan/random/random_mt19937.h

```diff
@@ -84,7 +84,7 @@ class Rng<MersenneTwister>
     { SEQAN_CHECKPOINT; }
 
     inline
-    unsigned
+    unsigned long
     operator()()
```

Comment:
You don't need to change anything in the random module because it will be removed entirely for 2.1.0.